### PR TITLE
Install packer on build agent - issue #280

### PR DESCRIPTION
**Pull request by nielsvbrecht**
Add code to the repository to install Packer on the build agents and solve issue #280

### CODE REVIEW

File: images/linux/scripts/installers/packer.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+
+#!/bin/bash
+
```
Comment (alepauly) [inline images/linux/scripts/installers/packer.sh]: nit: Please remove the extra blank lines.

**Comment by nielsvbrecht** [comment]
@alepauly Thanks for your feedback. I've removed multiple blank lines, but left some in for proper separation of functions. Hope this suffices!

**Comment by nielsvbrecht** [comment]
I was up very early so I added additional scripts to install Packer on Windows. I based it on the 7zip scripts except for the validation script with regards to separating out the version of Packer.

**Comment by maxim-lobanov** [comment]
Run CI builds. As soon as they are green, we can merge the PR

**Comment by nielsvbrecht** [comment]
> Run CI builds. As soon as they are green, we can merge the PR

We have green light ! ;)

**Pull request merged**